### PR TITLE
Add Kipluif

### PR DESCRIPTION
**Pull request by Spandamn**

### CODE REVIEW

File: data/mods/ssb/conditions.ts [hunk 1]
```diff
@@ -866,10 +866,22 @@ export const Conditions: {[k: string]: ModdedConditionData} = {
 			this.add(`c|${getName('KingSwordYT')}|Mucho texto`);
 		},
 		onSwitchOut() {
-			this.add(`c|${getName('KingSwordYT')}|Hasta la próximaaaa`);
+			this.add(`c|${getName('KingSwordYT')}| /teleport`);
```
Comment (KrisXV) [inline data/mods/ssb/conditions.ts]: This seems like a mistake

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -2390,6 +2390,52 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		type: "Dark",
 	},
 
+	// Kipluif
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
	// Kipkluif
```

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -2390,6 +2390,52 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+	kipup: {
+		accuracy: true,
+		basePower: 0,
+		category: "Status",
+		desc: "When used, if hit by an attack on the same turn this move was used, this Pokemon boost defense and special defense by 2 if the relevant stat is at 0 or lower, or 1 if the relevant stat is at +1 or higher, and increases priority of the next used move by 1.",
+		shortDesc: "If used and hit by an attack on the same turn, boosts defenses; boosts next moves priority by 1.",
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts:2399]: It’s called a shortDesc for a reason

File: data/mods/ssb/moves.ts [hunk 4]
```diff
@@ -2390,6 +2390,52 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+		},
+		condition: {
+			duration: 1,
+			onStart(pokemon) {
+				this.add('-message', 'This Pokémon prepares itself to be knocked down!');
+			},
+			onHit(pokemon, source, move) {
+				if (this.effectData.gotHit) return;
+				if (pokemon.side !== source.side && move.category !== 'Status') {
+					pokemon.volatiles['kipup'].gotHit = true;
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
					this.effectData.gotHit = true;
```

File: data/mods/ssb/moves.ts (hunk 3 above)
Comment (Spandamn) [reply data/mods/ssb/moves.ts:2399]: its... short?

File: data/mods/ssb/abilities.ts [hunk 5]
```diff
@@ -1135,6 +1135,35 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		gen: 8,
 	},
 
+	// Kipkluif
+	degenerator: {
+		shortDesc: "Opponents that switch out while this Pokémon is active lose 33% of their health.",
+		onStart(pokemon) {
```
Comment (KrisXV) [inline data/mods/ssb/abilities.ts:1141]: Does the side condition ever end?

File: data/mods/ssb/conditions.ts (hunk 1 above)
Comment (Spandamn) [reply data/mods/ssb/conditions.ts]: yes it is
ive commited and pushed, this prepush thingy is taking its sweet time

File: data/mods/ssb/abilities.ts (hunk 5 above)
Comment (Spandamn) [reply data/mods/ssb/abilities.ts:1141]: oops forgot that

**Comment by ghost** [comment]
looks gud

**Pull request merged**